### PR TITLE
docker-compose file

### PR DESCRIPTION
**Pull request by WesleyCharlesBlake**
Added docker-compose.yml for another option for running localstack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,12 @@ You can also spin up *LocalStack* without any installation requirements, using D
 docker run -it -p 4567-4580:4567-4580 -p 8080:8080 atlassianlabs/localstack
```
Comment (whummer) [inline README.md]: Can we change the port range to `4567-4581:4567-4581` here? (that was an oversight in a previous commit)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+version: '2'
+
+services:
+  localstack:
+    image: atlassianlabs/localstack
+    ports:
+      - "4567-4580:4567-4580"
```
Comment (whummer) [inline docker-compose.yml]: Same as above, please change the port range to `4567-4581:4567-4581`

**Comment by whummer** [comment]
Thanks for this PR! It might be worthwhile looking at the `docker-run` target in the `Makefile`. We have a bunch of configuration options there which probably should also be reflected in the `docker-compose.yml`.

For example, the volume mount `/var/run/docker.sock` is required for executing Lambda functions in separate Docker containers. We should also pass the configuration variables (`SERVICES`, `LAMBDA_EXECUTOR`, `KINESIS_ERROR_PROBABILITY`, etc) to the container using the `environment` configuration section (https://docs.docker.com/compose/environment-variables/#passing-environment-variables-through-to-containers)

**Comment by WesleyCharlesBlake** [comment]
thanks @whummer I will make some updates to cater for the confifuration.

File: README.md [hunk 3]
```diff
@@ -87,7 +87,13 @@ pip install localstack
...
+docker run -it -p 4567-4581:4567-4581 -p 8080:8080 atlassianlabs/localstack
+```
+
+Or with docker-compose:
+
+```
+docker-compose up
 ```
 
 ... or simply use the `make` target which runs the same command:
```
Comment (whummer) [inline README.md]: This is nit-picking, but: The statement in line 99 refers back to the command in line 90. Maybe we can move the new instructions for docker-compose below the `make docker-run` part?

**Comment by whummer** [comment]
@WesleyCharlesBlake did you have a chance to update the `docker-compose.yml` file to cater for the configuration? Thanks

**Pull request merged**